tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/mapper/BreakIteratorMapper.java

```diff
@@ -43,12 +43,16 @@
         specials.addValueAction("icu:boundaries/*", BreakIteratorMapper::addBoundary);
         specials.addValueAction(
             "icu:dictionaries/icu:dictionary", BreakIteratorMapper::addDictionary);
+        specials.addValueAction(
+            "icu:lstm/icu:lstmdata", BreakIteratorMapper::addLstmdata);
```

Comment:
What does this do and why did you add it now?

Reply:
See the email thread “Have you ever converted the lstm data to ICU4J” -- Peter went partway towards adding the lstm lines via the file generator, until we said that we don't want to add this data by default. I guess this is ok to keep for later.

Reply:
See my first comment. This was about adding support for the *right* way to include the lstm data when we decide to do that.

Reply:
See reply above.  I think we should separate the task of adopting the "right way" of adding the LSTM data from the task of updating to CLDR 40.